provider/testdata/data/uprops/sc.toml

```diff
@@ -0,0 +1,2436 @@
+# Copyright (C) 2021 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# file name: sc
+#
+# machine-generated by: upropdump.cpp
+
+[code_point_map.data]
```

Comment:
Nit: Please use this opportunity to update the data files to the new structure from ICU.  The layout of the file changed slightly based on review feedback.  The official tool is now checked in to ICU main.